### PR TITLE
Swiftmailer [MAILPOET-2363]

### PR DESCRIPTION
**Pull request by JanJakes**

### CODE REVIEW

File: lib/Config/RequirementsChecker.php [hunk 1]
```diff
@@ -17,9 +17,6 @@ class RequirementsChecker {
   public $vendor_classes = [
     '\ORM',
     '\Model',
-    '\Swift_Mailer',
-    '\Swift_SmtpTransport',
-    '\Swift_Message',
```
Comment (pavel-mailpoet) [inline lib/Config/RequirementsChecker.php]: Why did you remove those lines from here? The commit message says "Move Swiftmailer to vendor-prefixed" I don't see how it is relevant.

Comment (JanJakes) [reply lib/Config/RequirementsChecker.php]: @pavel-mailpoet It is, classes such as `\Swift_Mailer` won't exist anymore, it will be `MailPoetVendor\Swift_Mailer` but that one we don't need to check for conflicts.
